natgeo/mortar
# [SYTO-1015] First pass at handling rich text in pod and latest/most read components

https://jira.natgeo.com/browse/SYTO-1015

Updates for rich text compatibility on POD component as well as most read. Kicker logic was updated on most read to accommodate this.

To test:

- pull SYTO-1015
- npm test
- gulp serve
- verify fields are displaying properly on configurator pages as well as the components and content packages page.
